src/containerapp/azext_containerapp/custom.py

```diff
@@ -3359,3 +3360,43 @@ def containerapp_debug(cmd, resource_group_name, name, container=None, revision=
             if conn.is_connected:
                 logger.info("Caught KeyboardInterrupt. Sending ctrl+c to server")
                 conn.send(SSH_CTRL_C_MSG)
+
+
+def update_httprouteconfig(cmd, resource_group_name, name, http_route_config_name, yaml):
+    yaml_httprouteconfig = load_yaml_file(yaml)
+    # check if the type is dict
+    if not isinstance(yaml_httprouteconfig, dict):
+        raise ValidationError('Invalid YAML provided. Please see https://aka.ms/azure-container-apps-yaml for a valid YAML spec.')
```

Comment:
Seems this link doesn't contains the YAML example for httprouteconfig, right?

Reply:
Correct, the example hasn't been written yet. This link gets you to the docs where the example should be when it's ready.